ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-stable-4.18-upgrade-from-stable-4.17.yaml

```diff
@@ -47,26 +47,20 @@ releases:
       channel: candidate
       version: "4.17"
   arm64-target:
-    prerelease:
+    release:
       architecture: arm64
-      product: ocp
-      version_bounds:
-        lower: 4.18.0-0
-        stream: 4-dev-preview
-        upper: 4.19.0-0
+      channel: candidate
+      version: "4.18"
   latest:
     release:
       architecture: amd64
       channel: candidate
       version: "4.17"
   target:
-    prerelease:
+    release:
       architecture: amd64
-      product: ocp
-      version_bounds:
-        lower: 4.18.0-0
-        stream: 4-dev-preview
-        upper: 4.19.0-0
+      channel: candidate
+      version: "4.18"
```

Comment:
Do we have the EC build here for 4.18 already ?

Reply:
Yeah, refer to https://github.com/openshift/release/pull/56899#issuecomment-2359882809

Reply:
Ack. Good to know.